src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/AssessmentController.kt

```diff
@@ -155,9 +156,11 @@ class AssessmentController(
       is ValidatableActionResult.Success -> assessmentValidationResult.entity
     }
 
-    extractResultEntityOrThrow(
-      placementRequestService.createPlacementRequest(assessment, assessmentAcceptance.requirements)
-    )
+    if ((assessment.application as? ApprovedPremisesApplicationEntity)?.releaseDate != null) {
```

Comment:
I think we can use `assessmentAcceptance.requirements.expectedArrival` here, as this will be set to the release date. In fact, it may be worth us not sending any requirements if there is no arrival date.

Reply:
I think we need to persist this so we can use it to determine the status for the Application for 1405 too as we won't have the placement request data at that point? https://trello.com/c/pZz38nxr/1405-add-a-pending-status-to-an-application-assessment